renderer/components/Form/TextArea.js

```diff
@@ -37,16 +43,13 @@ const TextArea = ({
   ...rest
 }) => {
   const [hasFocus, setFocus] = useState(false)
-  const inputRef = useRef(forwardedRef)
   const [spaceProps, otherProps] = extractSpaceProps(rest)
   const { setValue, setTouched } = fieldApi
   const { value, maskedValue, error, asyncError } = fieldState
 
-  useEffect(() => {
-    if (willAutoFocus) {
-      inputRef.current.focus()
-    }
-  }, [willAutoFocus])
+  useLayoutEffect(() => {
```

Comment:
Could be an even better idea to extract the hook

Reply:
Good call. Updated.